package.json

```diff
@@ -14,25 +14,25 @@
   "scripts": {
     "test": "gulp test",
     "start": "node server/server.js",
-    "postinstall": "bower install && tsd install && gulp build"
+    "postinstall": "bower install && tsd update --save --overwrite && gulp build"
   },
   "dependencies": {
-    "rfc6902": "^1.0.8",
+    "bower": "1.4.2",
     "del": "2.0.2",
-    "bower": "1.4.2", 
-    "typescript": "1.6.2",
-    "tsd": "0.6.5", 
+    "express": "4.10.2",
     "gulp": "^3.9.0",
     "gulp-help": "^1.6.0",
     "gulp-inject": "3.0.0",
+    "gulp-rename": "^1.2.2",
     "gulp-sequence": "0.4.1",
     "gulp-tsconfig-files": "0.0.2",
     "gulp-tslint": "^3.1",
-    "express": "4.10.2",
-    "socket.io": "1.2.0"
-  },
-  "devDependencies": {
+    "rfc6902": "^1.0.8",
+    "socket.io": "1.2.0",
+    "tsd": "0.6.5",
+    "typescript": "1.6.2"
   },
+  "devDependencies": {},
```

Comment:
I have added some dependencies to `devDependencies` in order to allow it to build on Heroku, is there a reason to revert the change?